MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -48,13 +48,13 @@ - (instancetype)init {
   if ((self = [super init])) {
     _apiUrl = kMSDefaultApiUrl;
     _installUrl = kMSDefaultInstallUrl;
-    NSNumber *flag = [MS_USER_DEFAULTS objectForKey:kMSSDKHasLaunchedWithDistribute];
 
     /*
      * Delete API token if an application has been uninstalled and try to get a new one from server.
      * Under iOS 10.3, keychain data won't be automatically deleted by uninstall
```

Comment:
This comment is misleading "under 10.3" reads to me as if this was the case on 10.3.
Maybe say "for OS version < 10.3" or something.